src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx

```diff
@@ -1,18 +1,21 @@
 import { Button, IconSearch } from "hds-react";
-import React, { useState, useRef } from "react";
+import React, { useRef, useState } from "react";
 import { useRouter } from "next/router";
 import { gql, useLazyQuery } from "@apollo/client";
 import debounce from "lodash/debounce";
+import classNames from "classnames";
 
-import searchApolloClient from "../../../client/searchApolloClient";
-import { getUnifiedSearchLanguage } from "../../../client/utils";
 import queryPersister from "../../../util/queryPersister";
 import getURLSearchParamsFromAsPath from "../../../util/getURLSearchParamsFromAsPath";
 import Text from "../../text/Text";
 import SuggestionInput, {
   Suggestion,
 } from "../../suggestionInput/SuggestionInput";
 import styles from "./searchPageSearchForm.module.scss";
+import searchApolloClient from "../../../client/searchApolloClient";
+import { getUnifiedSearchLanguage } from "../../../client/utils";
+import { ShowMode } from "../searchHeader/SearchHeader";
+import getShowMode from "../../../util/getShowMode";
```

Comment:
To my eye it looks like these imports have been reordered in a weird way.

Reply:
There was a "little" merge conflict in the file. While resolving it the import order got messed up.